config/i18n/all-langs.ts

```diff
@@ -92,6 +94,20 @@ export const auditedCerts = {
     SuperBlocks.MachineLearningPy,
     SuperBlocks.CodingInterviewPrep,
     SuperBlocks.RelationalDb
+  ],
+  ukrainian: [
+    SuperBlocks.RespWebDesign,
+    SuperBlocks.JsAlgoDataStruct,
+    SuperBlocks.FrontEndDevLibs,
+    SuperBlocks.DataVis,
+    SuperBlocks.BackEndDevApis,
+    SuperBlocks.QualityAssurance,
+    SuperBlocks.SciCompPy,
+    // SuperBlocks.DataAnalysisPy,        // These are not yet approved atleast 95%
+    // SuperBlocks.InfoSec,               // These are not yet approved atleast 95%
```

Comment:
I think they can go in anyway, it's the importing of the project description to the curriculum that caused this reduction. It happened in all languages, and if the Ukrainian team didn't notice I think they can just been told and will be fast

Or the following certs need to wait too, as we want to have the certs published in order